pkg/transport/http/router/oauth2/oauth2.go

```diff
@@ -185,7 +187,9 @@ func (o *Auth) callback(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
-	setSessionCookie(w, token, claims)
+	if err := setSessionCookie(w, token, claims); err != nil {
+		logger.Error("Could not set session cookie", zap.Error(err))
```

Comment:
If we have a `logger` package it should not leak the underlying implementation (in this case zap). I propose, like in logr, to have `logger.Error` take `err error` as the first parameter which nil being valid/accepted.

Reply:
Removed them all.